### ISSUE TITLE
Need ability to remove chains

### ISSUE THREAD

**Issue by paulb39** [chainapsis/keplr-wallet#776]
**Is your feature request related to a problem? Please describe.**
Sometimes you need to remove a chain from the wallet and then re-connect it. Before Keplr 2.0 there was a little x next to the chains and you could click that to remove it. As far as I can tell this feature was removed in Keplr 2.0 and there's no way to do this anymore. 

**Describe the solution you'd like**
Add that little x back next to the chain like we had before- which would remove the chain from the wallet. 

**Describe alternatives you've considered**
N/A

**Additional context**
I am using the Chrome extension

**Comment by Thunnini** [comment]
You can remove chain on Settings -> General -> Manage Non-Native Chains
Functions are located in places that are intuitively hard to find. We're thinking about how to improve.

**Comment by paulb39** [comment]
@Thunnini we used to be able to remove any of them- I am trying to remove Kava and that is not in the list- is there no way to do that now? To improve you can revert back to how it was in v1- that was way better and I have no idea why you guys changed it.

**Comment by Thunnini** [comment]
From that menu you can only delete suggested chains from external websites. Perhaps what you want is in "Manage Chain Visibility"

**Comment by paulb39** [comment]
No, I am looking for the remove from wallet function we had in v1- the problem is sometimes when a coin does an update we need to remove and re-add from the wallet. Using the manage coin visibility doesn't actually disconnect it from the wallet, so when I go to https://app.kava.io it doesn't ask me to re-connect it to the wallet when I remove visibility.
